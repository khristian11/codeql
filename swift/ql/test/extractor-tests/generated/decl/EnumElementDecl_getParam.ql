import codeql.swift.elements
import TestUtils

from EnumElementDecl x, int index
where toBeTested(x)
select x, index, x.getParam(index)
