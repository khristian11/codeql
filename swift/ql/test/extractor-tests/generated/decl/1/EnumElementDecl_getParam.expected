| declarations.swift:19:10:19:22 | nodata1 | 0 | declarations.swift:19:18:19:18 | _ |
| declarations.swift:20:10:20:21 | intdata | 0 | declarations.swift:20:18:20:18 | _ |
| declarations.swift:21:10:21:35 | tuple | 0 | declarations.swift:21:16:21:16 | _ |
| declarations.swift:21:10:21:35 | tuple | 1 | declarations.swift:21:21:21:21 | _ |
| declarations.swift:21:10:21:35 | tuple | 2 | declarations.swift:21:29:21:29 | _ |
